src/frontend/pages/game.tsx

```diff
@@ -97,13 +160,8 @@ export default function Home(): ReactElement {
         </div>
 
         <div className={styles.gameDisplay}>
-          <GameTable takeVal={takeVal} />
-          <Timer
-            time={TIMER_SECONDS}
-            shouldResetTimer
-            onTimerOver={() => setTakeVal(Math.floor(Math.random() * 11))}
-            customClass={styles.gameTimer}
-          />
+          <GameTable pointsRemaining={pointsRemaining} />
+          <div className={`${styles.timer}`}>{timeLeft.toString()}</div>
```

Comment:
Out of curiosity does it matter if we do {timeLeft} vs {timeLeft.toString()}. Isn't TimeLeft just a number

Reply:
Yeah you're right! We could just display the number